src/main/java/seedu/address/logic/commands/wallet/WalletExpenseCommand.java

```diff
@@ -32,7 +32,7 @@
             + PREFIX_DATE + "10/10/2010 "
             + PREFIX_TAG + "Food";
 
-    public static final String MESSAGE_SUCCESS = "New expense added: %1$s";
+    public static final String MESSAGE_SUCCESS = "New expense added: %1$s\nYour expenditure this month is: %2$s/%3$s";
```

Comment:
Maybe it should specify the month rather than 'this month' because the user might not be adding the expense during the same month